websites/D/DiscordLink/presence.ts

```diff
@@ -0,0 +1,34 @@
+const presence = new Presence({
+  clientId: "627875524298932254"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo-dlm"
+  };
+
+  const browsingStamp = Math.floor(Date.now() / 1000);
+  presenceData.startTimestamp = browsingStamp;
+
+  if (window.location.pathname.endsWith("/")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Home Page";
+  } else if (window.location.pathname.endsWith("/add")) {
+    presenceData.details = "Adding a server";
+  } else if (window.location.pathname.endsWith("/view")) {
+    presenceData.details = "Viewing a server";
+  } else if (window.location.pathname.startsWith("/edit")) {
+    presenceData.details = "Editing a server";
```

Comment:
maybe add servername/more details?

Reply:
Ok

Reply:
presence that doesnt do much than this will get rejected, always add more info like in this case:
details: Viewing a server
state: <server name>

fyi: there is also a i18n implementation which you can get via presence.getStrings method (you can read more on premid's documentation)
all strings: https://api.premid.app/v2/langFile/presence/en